glossary.md

```diff
@@ -10,51 +10,49 @@ This page contains a selection of key terms important to understanding the struc
 
 ## Term List
 
-- **[Cohort](http://browser.datacommons.org/kg?dcid=CohortSet)**
+### **[Cohort](https://datacommons.org/browser/CohortSet)**
```

Comment:
the bold might not be necessary if it is now also a heading.

Reply:
should we make the links clearer by making it separate from the heading? e.g.

### Cohort ([graph node](https://datacommons.org/browser/CohortSet))

or

### Cohort 
 ([see the browser node](https://datacommons.org/browser/CohortSet))

Reply:
Good idea. Updated